plugins/fortinet_fortipam.rb

```diff
@@ -0,0 +1,17 @@
+Plugin.define do
+  name "fortinet fortipam"
+  authors [
+    "Ostorlab",
+  ]
+  version "0.1"
+  description "FortiPAM provides tightly controlled privileged access to the most sensitive resources within an organization. "
+  website "https://www.fortinet.com/products/fortipam"
+
+  matches [
+    {
+      :search => "body",
+      :regexp => /FortiPAM/,
+      :name => "FortiPAM Body Content"
+    }
```

Comment:
wouldn't this give FP , like if we give it a page with Fortipam in the body it will deduce it's using the it , like for example https://www.fortinet.com/products/fortipam

Reply:
Your link is not working. And that's the indicator that I found useful.